src/app/components/source/UserPrivacy.js

```diff
@@ -210,6 +214,22 @@ class UserPrivacy extends Component {
             />
           </CardText>
         </Card>
+        <h2 style={style}>
+          <FormattedMessage id="userPrivacy.connected_accounts" defaultMessage="Connected accounts" />
+        </h2>
+        <Card style={cardStyle}>
+          <CardText style={cardTextStyle}>
+            <List>
+              { providers.map(provider => (
+                <UserConnectedAccount
+                  provider={provider}
+                  user={user}
+                  key={provider.key}
```

Comment:
If we support multiple accounts from the same provider and user, this `key` won't be unique